roles/container-engine/docker/vars/redhat-7.yml

```diff
@@ -0,0 +1,39 @@
+---
+# containerd versions are only relevant for docker
+containerd_versioned_pkg:
```

Comment:
Since you are not changing these dictionaries in the PR I don't think they need to be included in this override. Only the `docker_versioned_pkg` and `docker_cli_versioned_pkg` should be included because they override the ones which include the epoch that work on el8.

Reply:
Since the gather facts tasks uses `with_first_found` only the first match will be included. In fact there is not an override, because `containerd_versioned_pkg`,  `docker_cli_versioned_pkg` and `containerd_versioned_pkg` are not set by default, they are included from one of the `roles/container-engine/docker/vars` files.

So if we do not set also `containerd_versioned_pkg` variable in `redhat-7.yml` vars file it won't be defined, won't it?